runtime/src/main/java/com/redhat/ipaas/rest/Connections.java

```diff
@@ -85,8 +93,50 @@ public void update(
     public void delete(
         @ApiParam(value = "id of the connection", required = true) @PathParam("id") String id) {
         dataMgr.delete(Connection.class, id);
-
     }
-
-
+    
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path(value = "/{id}/collection/tags")
```

Comment:
That seems to be a dead spec (last updated 2 years ago) & doesn't seem to have caught on. Most real world REST APIs don't have this segment which seems redundant & doesn't fulfil any purpose other than making the URL longer. `/{id}/tags` is enough & is in much more common usage, a de facto standard I would say.